src/com/google/javascript/jscomp/js/es6/util/inherits.js

```diff
@@ -58,14 +58,17 @@ $jscomp.inherits = function(childCtor, parentCtor) {
   /** @override */
     childCtor.prototype.constructor = childCtor;
 
-  for (var p in parentCtor) {
-    if (Object.defineProperties) {
-      var descriptor = Object.getOwnPropertyDescriptor(parentCtor, p);
-      if (descriptor) {
-        Object.defineProperty(childCtor, p, descriptor);
+  if (Object.defineProperties) {
+    var propNames = Object.getOwnPropertyNames(parentCtor);
+    for (var i = 0; i < propNames.length; i++) {
+      var descriptor = Object.getOwnPropertyDescriptor(parentCtor, propNames[i]);
```

Comment:
I may be wrong about this. When I added the 'require' locally to see if it would fix our failing internal test it just added more errors about stuff defined in getownpropertysymbols.js.
Perhaps just adding to the explict externs in the test classes as you've done is the way for me to fix that.

@shicks how do 'require' lines work in these files?
Should they all explicitly require anything they use from other file,
or is that not needed for things declared in externs?